charts/portworx/templates/portworx-controller.yaml

```diff
@@ -1,5 +1,5 @@
 {{- $AKSorEKSInstall := or .Values.AKSInstall  .Values.EKSInstall  }}
-{{- if or ((.Values.openshiftInstall) and (eq .Values.openshiftInstall true)) ($AKSorEKSInstall) ((.Capabilities.KubeVersion.GitVersion | regexMatch "gke")) }}
+{{- if or (and (.Values.openshiftInstall) (eq .Values.openshiftInstall true)) ($AKSorEKSInstall) ((.Capabilities.KubeVersion.GitVersion | regexMatch "gke")) }}
```

Comment:
I don't know if this works as expected or not.

Reply:
@pault84 those changes are needed for helm 3.3. I tested with helm version 3.3, 3.2.4 and 2.16.10